app/javascript/app/page/page.jsx

```diff
@@ -3,12 +3,15 @@ import Proptypes from 'prop-types';
 
 import TopBar from 'components/topbar';
 import Nav from 'components/nav';
-import styles from './page-styles.scss';
+import Header from 'components/header';
+
+import styles from "./page-styles.scss"; // eslint-disable-line
 
 const Layout = ({ children }) =>
   (<div>
     <TopBar />
     <Nav />
+    <Header />
```

Comment:
I don't think that this component should be listed in the layout level because each page has its own Header... so I'd move it to the page.